### PR TITLE
Fixes #10732: Remove attributes from user and use admin for remote ca…

### PR DESCRIPTION
**Pull request by ehelms**
…lls.

### CODE REVIEW

File: app/controllers/katello/dashboard_controller.rb [hunk 1]
```diff
@@ -65,26 +59,9 @@ def subscriptions_totals
...
-      user_preferences[:dashboard] = {} unless user_preferences.key? :dashboard
-      user_preferences[:dashboard][key] = value
-      current_user.preferences = user_preferences
-      current_user.save!
-    end
-
     helper_method :quantity
     def quantity
-      current_user.preferences_hash[:dashboard][params[:action]][:page_size] rescue 5
+      5
```
Comment (jlsherrill) [inline app/controllers/katello/dashboard_controller.rb]: why not use the app-wide page size for this?

File: app/models/katello/concerns/user_extensions.rb [hunk 2]
```diff
@@ -58,113 +33,9 @@ def cp_oauth_header
...
-        #set the default org if it's an actual org_id
-        def default_org=(org_id)
-          self.preferences_hash[:user] = { } unless self.preferences_hash.key? :user
-          if !org_id.nil? && org_id != "nil"
-            organization = Organization.find_by_id(org_id)
-            self.preferences_hash[:user][:default_org] = organization.id
-          else
-            self.preferences_hash[:user][:default_org] = nil
-          end
-        end
```
Comment (jlsherrill) [inline app/models/katello/concerns/user_extensions.rb]: can we really get rid of default_org?

Comment (jlsherrill) [inline app/models/katello/concerns/user_extensions.rb]: ohhh that is in foreman now

File: app/models/katello/concerns/user_extensions.rb [hunk 3]
```diff
@@ -58,113 +33,9 @@ def cp_oauth_header
...
-
-        def subscriptions_match_system_preference
-          self.preferences_hash[:user][:subscriptions_match_system] rescue false
```
Comment (jlsherrill) [inline app/models/katello/concerns/user_extensions.rb]: any idea what this was supposed to do?

Comment (jlsherrill) [inline app/models/katello/concerns/user_extensions.rb]: any of these 'subscriptions_match_*' prefs?

File: db/migrate/20150603045418_remove_user_fields.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+class RemoveUserFields < ActiveRecord::Migration
+  def up
+    remove_column :users, :helptips_enabled
```
Comment (jlsherrill) [inline db/migrate/20150603045418_remove_user_fields.rb]: Bye bye helptips :wave: we hardly knew ye

File: app/models/katello/concerns/user_extensions.rb (hunk 3 above)
Comment (ehelms) [inline app/models/katello/concerns/user_extensions.rb]: I do not recall, I could dig through the old code but these aren't used anywhere that I could find.

File: app/controllers/katello/dashboard_controller.rb (hunk 1 above)
Comment (ehelms) [inline app/controllers/katello/dashboard_controller.rb]: Good call - will update.

**Comment by ehelms** [comment]
Changed the dashboard controller to use entries_per_page and updated the seeds.rb to remove a requires I missed. This, from my testing, also fixes an issue with katello:reset that was introduced with the backend check work.

**Comment by jlsherrill** [comment]
Tested and worked well.  created product, repo, published content view.  Registered a client, subscribed checked bound repos,and checked that applicability generation worked.

ACK

**Pull request merged**